src/app/home/navbar/navbar.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+import { User } from 'src/app/reducers/auth/auth.reducer';
+import { HomeService } from '../home.service';
+
+@Component({
+  selector: 'app-navbar',
+  templateUrl: './navbar.component.html',
+  styleUrls: ['./navbar.component.scss'],
+})
+export class NavbarComponent {
+  @Input() user: User | null = null;
+  @Input() isLoggedIn: boolean | null = null;
```

Comment:
Maybe for the primitive

```suggestion
  @Input() isLoggedIn: boolean = false;
```

?